pkg/solver/solver.go

```diff
@@ -13,11 +13,26 @@
 package solver
```

Comment:
What is the responsibility of `solver` package and this `solver.go` ?

Reply:
Added a `doc.go` in the package with a short description of what it's for.